app/controllers/concerns/api/v1/rescuable.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Rescuable
+      extend ActiveSupport::Concern
+
+      included do
+        rescue_from ActiveRecord::RecordNotFound do
+          render_errors(
+            details: [I18n.t('api.errors.not_found')],
+            status: :not_found
+          )
+        end
+      end
```

Comment:
```suggestion
      included do
        rescue_from ActiveRecord::RecordNotFound { render_not_found_error }
      end
      
      def render_not_found_error
        render_errors(
          details: [I18n.t('api.errors.not_found')],
          status: :not_found
        )
      end
```

The included bock is here to gather the DSL specific to rails. It is recommended to keep it short so that we can easily grasp how the concern is affecting the class that will implement it.

Reply:
I was not able to get `rescue_from ActiveRecord::RecordNotFound { render_not_found_error }` working with `missing end` error so I modified this to
```
        rescue_from ActiveRecord::RecordNotFound do
          render_not_found_error
        end
```

Reply:
Hmm... probably due to the DSL...
Let's see what the documentation says about it... 👀 

https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from

oh! 
Can you try this?

```
rescue_from, :with => render_not_found_error
```

🚀

Reply:
```
rescue_from, with: render_not_found_error
```

Maybe this works too... need to try :D

Reply:
Works great 👍